k8s/overlays/rbb-crc/kustomization.yaml

```diff
@@ -1,7 +1,7 @@
 secretGenerator:
 - name: basic-auth
   namespace: onboarding
-  behavior: create  
+  behavior: create
   literals:
   - users=admin pass
 bases:
```

Comment:
doesn't this need to be changed to generate the the correct form of the secret - or am I missing something here?

Reply:
I didn't touch the `rbb-crc` directory in this change. I was going to ask if we could remove it from the repository (just maintain it locally if you need something different from the `crc` overlay, or make the `crc` repository suitable for your needs). What do you think?

Reply:
I put it under source control as kristi was running 'crc' without basic authentication but with a secure connection, and I was running it with basic authentication and  without a secure connection.